pkg/lib/serverTime.js

```diff
@@ -321,13 +316,166 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
+
+    function set_ntp_config_timesyncd(config) {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230113-125341-c8c3ecb6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)